app/services/import_tag_service.rb

```diff
@@ -0,0 +1,22 @@
+class ImportTagService < ApplicationService
+  def initialize(post)
+   @post = post
+  end
+
+  def call
+    tags = post.message.scan(/(?:\s|^)(?:#(?!\d+(?:\s|$)))(\w+)(?=\s|$)/i).flatten.map(&:downcase)
```

Comment:
the `\w` matcher will match only the english characters correct? should we also match arabic characters or should we move it to future changes after we're done with this?

Reply:
yeah , this regex is a black box for me right now :blush:  , i just used it from Stackoverflow , so i think we can check it again later